tendrl/node_agent/discovery_modules/plugin.py

```diff
@@ -0,0 +1,11 @@
+class PluginMount(type):
```

Comment:
Dont re-define. I already have a commons util defined for this at https://github.com/Tendrl/commons/pull/102. Use the same.